### PR TITLE
completed task 5 GET /api/articles

### PR DESCRIPTION
**Pull request by lalauo**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -87,23 +88,52 @@ describe("GET /api/articles/:article_id", () => {
...
       .expect(200)
-      .then(({ body }) => {
-        console.log(body);
+      .then(({ body: { articles } }) => {
+        if (articles.length > 1) {
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("author");
```
Comment (pippaamy) [inline __tests__/app.test.js:101]: I would like to see the type of the values as well as the properties  eg title will be a string etc

File: __tests__/app.test.js [hunk 2]
```diff
@@ -87,23 +88,52 @@ describe("GET /api/articles/:article_id", () => {
...
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles[0]).toEqual({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          created_at: "2020-11-03T09:12:00.000Z",
```
Comment (pippaamy) [inline __tests__/app.test.js:120]: we can use jest sorted for this :)

**Review by pippaamy — COMMENTED** [review]
Great job - some comments to have a think about :)

File: __tests__/app.test.js (hunk 1 above)
Comment (lalauo) [reply __tests__/app.test.js:101]: Noted!

File: __tests__/app.test.js (hunk 2 above)
Comment (lalauo) [reply __tests__/app.test.js:120]: Totally forgot about that! Thanks for the reminder

**Pull request merged**